### PR TITLE
Mosaico/fix fallback images

### PR DESCRIPTION
**Pull request by mrskug**

### CODE REVIEW

File: apps/mosaico/src/Mosaico/FallbackImage.purs [hunk 1]
```diff
@@ -0,0 +1,9 @@
...
...AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAH//2Q=="
```
Comment (vapaj) [inline apps/mosaico/src/Mosaico/FallbackImage.purs:6]: What was the reason you wanted to use inline images for these and not a cdn? I can come up with two reasons: these won't most likely ever change and they are quite small in size. Anything else?

File: apps/mosaico/src/Mosaico/FallbackImage.purs [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
```
Comment (vapaj) [inline apps/mosaico/src/Mosaico/FallbackImage.purs:9]: As this string is quite a handful, I suggest we change this to `fallbackImage _ = fallbackImage HBL` 😁

File: apps/mosaico/src/Mosaico/FallbackImage.purs (hunk 1 above)
Comment (mrskug) [reply apps/mosaico/src/Mosaico/FallbackImage.purs:6]: nope

File: apps/mosaico/src/Mosaico/FallbackImage.purs (hunk 2 above)
Comment (mrskug) [reply apps/mosaico/src/Mosaico/FallbackImage.purs:9]: ah yes, must have been a monday :grin:

File: apps/mosaico/src/Mosaico/FallbackImage.purs (hunk 1 above)
Comment (vapaj) [reply apps/mosaico/src/Mosaico/FallbackImage.purs:6]: Fair enough :D

**Review by vapaj — APPROVED** [review]

**Pull request merged**